public/reference/accounting-faculty.html

```diff
@@ -227,7 +227,7 @@ <h5>Research Interests</h5>
                   </div>
                   <div id="collapseOne" class="collapse" data-bs-parent="#accordion">
                   <div class="card-body">
-                  <div class="accordian-body-cntnt">
+                  <div class="accordian-body-content">

```

Comment:
should this accordi**o**n?

Reply:
on second inspection, we should probably keep public/reference/* untouched as possible